lighthouse-core/report/handlebar-helpers.js

```diff
@@ -63,10 +63,9 @@ const handlebarHelpers = {
     return calculateRating(totalScore);
   },
 
-  // Converts an aggregation's score to a rating used for styling.
+  // Converts an aggregation's score to a pass/fail rating.
   getAggregationScoreRating: score => {
-    const rating = calculateRating(score * 100);
-    return rating !== 'good' ? 'poor' : rating; // avg rating maps to failure for aggregation.
+    return score === 1 ? 'good' : 'poor';
```

Comment:
Pics!

All audits are 100/100:

<img width="600" alt="screen shot 2017-02-21 at 1 35 16 pm" src="https://cloud.githubusercontent.com/assets/238208/23190197/993de258-f83a-11e6-9f29-4d0620bc390c.png">

Some audits are not 100/100:

<img width="758" alt="screen shot 2017-02-21 at 1 35 11 pm" src="https://cloud.githubusercontent.com/assets/238208/23190208/a811a6ca-f83a-11e6-9057-71a0fc35a9bf.png">